org.eclipse.xtext/pom.xml

```diff
@@ -0,0 +1,68 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.xtext</groupId>
+		<artifactId>xtext-core</artifactId>
+		<version>2.30.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.xtext</artifactId>
+	<packaging>eclipse-plugin</packaging>
+
+	<name>Xtext Runtime Bundle</name>
+	<description>Core component of Xtext.</description>
+
+	<dependencies>
+		<dependency>
```

Comment:
here optional stuff is missing too

Reply:
So optional dependencies should now be handled correctly